trap.go

```diff
@@ -188,6 +188,26 @@ func (t *TrapListener) Listen(addr string) error {
 			if traps != nil {
 				t.OnNewTrap(traps, remote)
 			}
+			if traps.PDUType == InformRequest {
+				traps.PDUType = GetResponse
+				log.Printf("sending INFORM response: %+v\n", traps)
+
+				ob, err := traps.marshalMsg()
+				if err != nil {
+					return fmt.Errorf("Error marshaling INFORM response: %v\n", err)
+				}
+
+				// Send the return packet back.
+				count, err := conn.WriteTo(ob, remote)
+				if err != nil {
+					return fmt.Errorf("Error sending INFORM response: %v\n", err)
+				}
+
+				// This isn't fatal, but should be logged.
+				if count != len(ob) {
+					log.Printf("Failed to send all bytes of INFORM response!\n")
```

Comment:
Use `t.Params.logPrintf` so that the library user can control the output.

Reply:
Ah, yeah, I had missed this behavior elsewhere, thanks.